src/Hesap.java

```diff
@@ -0,0 +1,23 @@
+public class Hesap {
+    Zemin zemin = new Zemin();
```

Comment:
Değişkenleri new ile tanıtmak  hafızada o değişken  için yeni bir yer açar.

Reply:
Siz yazınca fark ettim; task de nesne oluşturun denmemiş, değişken oluşturun denmiş. Düzelttim.

Zaman ayırıp tek tek inceleyip, geri dönüş verdiğiniz için teşekkür ediyorum.